test-data/unit/check-38.test

```diff
@@ -0,0 +1,21 @@
+[case testScopedIgnore]
+def f(a: int): ...
+f(
+    'BAD'
+)  # type: ignore
```

Comment:
I would add few more tests with expressions that are not function calls. For example list and dictionary displays, operators (like + and *), etc.